mod/settings.php

```diff
@@ -1193,7 +1181,7 @@ function settings_content(App $a)
 		'$nickname_block' => $prof_addr,
 
 		'$h_pass' 	=> L10n::t('Password Settings'),
-		'$password1'=> ['password', L10n::t('New Password:'), '', ''],
+		'$password1'=> ['password', L10n::t('New Password:'), '', L10n::t('Allowed characters are a-z, A-Z, 0-9 and special characters except white spaces and colon (:).')],
```

Comment:
Below you write *can\'t contain accentuated letters*, which for me are special characters that you allow here.

Reply:
Here are the allowed characters:
- `a-z`
- `A-Z`
- `0-9`
- ``!"#$%&'()*+,-./;<=>?@[\]^_`{|}~``

Reply:
Why are characters with accents not allowed? Are characters outside of ASCII which are not letters or letters without accents not in Latin (like ‘♭’, ‘Ψ’, ‘—’, ‘⭾’) allowed? The instructions in the form to change password are very unclear.

Reply:
It has to do with our use of HTTP Basic Authentication, see https://github.com/friendica/friendica/issues/6355#issuecomment-450608448

Reply:
If I understand that correctly, characters outside of ASCII are not allowed. So that should be noted in the form to change password, because I almost had a problem with that.

Reply:
What is unclear in "Allowed characters are a-z, A-Z, 0-9 and special characters except white spaces and accentuated letters. Password length is limited to 72 characters." ?

Reply:
That characters outside of ASCII are not allowed is unclear. ‘♭’, ‘—’, ‘⭾’ are special characters, so they fulfill the instruction, but they are not allowed IIUC.

Reply:
Got it. These can't be easily typed with most keyboards, so I didn't think it was relevant to explicitly exclude them. And "ASCII" may not mean anything for most people we expect to use Friendica either.